### PR TITLE
Add doc for PrometheusMetricsServlet

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

File: docs/en/operation/Metrics-System.md [hunk 1]
```diff
@@ -96,6 +97,43 @@ The filename will correspond with the metric name.
...
+[Prometheus](https://prometheus.io/) is a monitoring tool that can help monitoring Alluxio metrics changes.
+
+In the metrics property file, `$ALLUXIO_HOME/conf/metrics.properties` by default, add the following properties:
+
+```
+# Enable PrometheusMetricsServlet
+sink.prometheus.class=alluxio.metrics.sink.PrometheusMetricsServlet
+```
+
+You can send an HTTP request to `/metrics/prometheus/` of the Alluxio leading master or workers to get a snapshot of metrics in Prometheus format.
```
Comment (Xenorith) [inline docs/en/operation/Metrics-System.md:111]: a HTTP request to the `/metrics/prometheus/` endpoint of the Alluxio leading master or workers

File: docs/en/operation/Metrics-System.md [hunk 2]
```diff
@@ -96,6 +97,43 @@ The filename will correspond with the metric name.
 
 Refer to `metrics.properties.template` for all possible sink specific configurations. 
 
+### Prometheus Sink Setup
+
+[Prometheus](https://prometheus.io/) is a monitoring tool that can help monitoring Alluxio metrics changes.
+
+In the metrics property file, `$ALLUXIO_HOME/conf/metrics.properties` by default, add the following properties:
```
Comment (Xenorith) [inline docs/en/operation/Metrics-System.md:104]: does the server need to be restarted after setting this property for it to work?

Comment (LuQQiu) [reply docs/en/operation/Metrics-System.md:104]: Added the distributed and server restarted part.

File: docs/en/operation/Metrics-System.md (hunk 1 above)
Comment (LuQQiu) [reply docs/en/operation/Metrics-System.md]: Got it! thanks

File: docs/en/operation/Metrics-System.md [hunk 3]
```diff
@@ -96,6 +97,46 @@ The filename will correspond with the metric name.
...
+
+```
+# Enable PrometheusMetricsServlet
+sink.prometheus.class=alluxio.metrics.sink.PrometheusMetricsServlet
+```
+
+If Alluxio is deployed in a cluster, this file needs to be distributed to all the nodes.
+Restart the Alluxio servers to active new configuration changes.
```
Comment (Xenorith) [inline docs/en/operation/Metrics-System.md:112]: to activate the new configuration changes

Comment (Xenorith) [reply docs/en/operation/Metrics-System.md:112]: oh jeez the incorrect wording was in more than one place.... >.<

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**